### DISCUSSION TITLE
PR open for six months without comment or being accepted

### DISCUSSION THREAD

**Discussion by mudnug** [sipsorcery-org/sipsorcery#919] [General]
Completing the Pull Request at https://github.com/sipsorcery-org/SIPSorceryMedia.Encoders/pulls and pushing to nuget would allow later versions of sipsorcery to be used more easily. 

If there are reasons why the PR can't be completed, please indicate on the PR

**Comment by mudnug** [comment]
Thank you for completing the PR.

 Is the workflow to release the build to nuget something that requires revving the version number? I see appveyor ran

`dotnet pack src\SIPSorceryMedia.Encoders.csproj -p:NuspecFile=SIPSorceryMedia.Encoders.nuspec -c Release --no-build`

and it looks like the 

`src\bin\Release\SIPSorceryMedia.Encoders.0.0.10-pre.nupkg` was generated, which has the same version number as the last release.

**Reply by sipsorcery** [reply]
https://www.nuget.org/packages/SIPSorceryMedia.Encoders/0.0.12-pre

**Comment by mudnug** [comment]
Thanks again.
